modules/i3s/test/i3s-loader.spec.js

```diff
@@ -33,7 +33,10 @@ test('I3SLoader#Load tile content', async t => {
 });
 
 test('I3SLoader#DRACO geometry', async t => {
-  const content = await loadI3STileContent({i3s: {useDracoGeometry: true}});
+  const content = await loadI3STileContent({
+    i3s: {useDracoGeometry: true},
+    fetchOptions: {worker: false}
```

Comment:
Doesn't parse already support `options.fetch` and `options.worker`?

Reply:
Tile3D class uses `fetchOptions` sub-object when loads content https://github.com/visgl/loaders.gl/blob/d37a7e0e4ffe3b3ef2f939422736214dbc136aff/modules/tiles/src/tileset/tile-3d.js#L208